### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [claredickerson/prj-rev-bwfs-tea-cozy#4]
Each image should have a descriptive alt text. The visually impaired uses screen readers to read this out loud so they can know what the picture is about.

**Comment by claredickerson** [comment]
This seems like a bit of a nit-pick since I literally never remember this coming up in any other exercise, except maybe mentioned when we learned how to link images...but sure, ok. lol

**Comment by farishkash** [comment]
It isn't and it is a nit pick.   As I stated in my summary it more about best practices.

Here is the actual problem is the program talks about accessibility later but some of the reviewers are pushing this now as a best practice.  There are a lot more visually impaired or blind people who use the web and unless you know one you might not be aware of it.  

For example people don't realize that the iPhone will read out a whole webpage using alt text and ARIA roles so the person can do something such get the phone number from a website and call it.

Also some companies are expecting that you do this if you are job interviewing with them so I always mention it.  

I don't know your motivation to take this course so the best thing I can do is assume you are planning to get a job so I am trying to put you in the best position possible.
